app/helpers/injection_helper.rb

```diff
@@ -10,6 +10,14 @@ def inject_enterprises(enterprises = Enterprise.activated.includes(address: :sta
     )
   end
 
+  def inject_map_pointers()
+    inject_json_ams(
+        'enterprises',
+        Enterprise.activated.includes(address: :state).all,
+        Api::ShopsSerializer
+    )
+  end
```

Comment:
we can keep the syntax as in other methods, right?

```suggestion
def inject_map_pointers
   inject_json_ams "enterprises", Enterprise.activated.includes(address: :state).all, Api::ShopsSerializer
end
```

Reply:
I was trying to keep the line under 100 characters, but I think that cop is disabled in injection_helper, so I could make it one line. I think I prefer it that way too, tbh. :+1: